module-cheatsheets/RNA-seq-cheatsheet.md

```diff
@@ -88,7 +88,7 @@ Read the Salmon tool documentation [**here**](https://salmon.readthedocs.io/en/l
 
 ### Useful command line commands:
 <br>
-![Useful Command Line Commands](diagrams/command_line_example.jpeg)
+![Useful Command Line Commands](diagrams/command-line-example.jpeg)
```

Comment:
On master, this is not coming up for me in the Markdown file (at least on my phone): https://github.com/AlexsLemonade/training-modules/blob/master/module_cheatsheets/Bulk_RNA-seq_Cheatsheet.md#useful-command-line-commands

I would look into that a bit, especially if you will be linking this in the README.